Formula/mypy.rb

```diff
@@ -2,8 +2,8 @@ class Mypy < Formula
   desc "Experimental optional static type checker for Python"
   homepage "http://www.mypy-lang.org/"
   url "https://github.com/JukkaL/mypy.git",
-    :tag => "v0.4.2",
-    :revision => "b8a2df91707bc3d5c9c3677c8017b614e41d017a"
+      :tag => "v0.4.3",
```

Comment:
I do prefer the equal indent, whether the `:` lines up with `url` or lines up with the start of the `"<url>` itself, but I'm not particularly passionate about it.

The `go_resource`s tend to differ depending on whether they were written by a human or by `gdm`, heh.